### ISSUE TITLE
Direct editing issue on 25

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/text#3121]
Seems something is broken with direct editing on iOS and Android with 25:

<img src="https://user-images.githubusercontent.com/3404133/193809195-944806f3-ce17-4050-b829-2e3b7417d70e.png" width="300" />

**Comment by juliushaertl** [comment]
Seems to work through the browser but the new UI is also creating an issue there:

<img width="1680" alt="Screenshot 2022-10-04 at 13 44 11" src="https://user-images.githubusercontent.com/3404133/193811327-46273e5a-46b1-4cbd-a012-d8dd61b23591.png">

**Comment by juliushaertl** [comment]
```
curl --request POST \
  --url 'http://admin:admin@nextcloud.dev.local/ocs/v2.php/apps/files/api/v1/directEditing/open?editorId=text&creatorId=textdocument&format=json' \
  --header 'Content-Type: multipart/form-data' \
  --header 'OCS-APIRequest: true' \
  --form path=/Readme1.md
```

**Comment by mejo-** [comment]
Not sure whether this is fixed by #3148. Someone with a mobile device connected to their test environment would need to check.
